### DISCUSSION TITLE
If the API documentation has basic authorization

### DISCUSSION THREAD

**Discussion by margaritluch** [apideck-libraries/portman#565] [Q&A] [answered]
My API documentation on swagger has a basic authorization. Does Portman support it?
I know it is possible to send query parameters inside the link but I'm not sure whether its possible to send a basic authentication header.

**Comment by thim81** [comment]
Hi @margaritluch 

The CLI does a simple fetch to remotely get the file. 

The basic auth that protects the OpenAPI file will prevent that. 

You could write a simple bash or node script to download the file first using with the basic auth and next pass the downloaded file to Portman for further processing.

**Comment by margaritluch** [comment, marked as answer]
HI @thim81 
Thank you for your answer. I ended up making a node script that downloads file locally then sends to portman for processing and it works fine, thank you :)
